app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -140,6 +140,7 @@ export const ProjectBoard = ({
         onClose={closeHistory}
         title={`${I18n.t('projects.show.history')} '${history.storyTitle}'`}
         data-id="history-column"
+        canCloseColumn
```

Comment:
Since this component name is `Column`, I think you could rename this prop to `canClose` 🤔

Reply:
Yes, it's a good idea, thank you.